ts-rs/tests/raw_idents.rs

```diff
@@ -1,8 +1,10 @@
+#![allow(non_camel_case_types, dead_code)]
+
 use ts_rs::TS;
 
-#[allow(non_camel_case_types, dead_code)]
 #[derive(TS)]
-struct r#enum {
+#[ts(export, export_to = "tests-out/raw_idents/")]
+struct r#struct {
```

Comment:
Had to rename `r#enum` to `r#struct` because `type enum = ...` is not valid TypeScript